package.json

```diff
@@ -67,17 +68,20 @@
     "npm-run-all": "^4.1.3",
     "prettier": "^1.14.2",
     "rimraf": "^2.6.2",
-    "rollup": "^0.64.1",
+    "rollup": "^0.65.2",
     "rollup-plugin-alias": "^1.4.0",
-    "rollup-plugin-babel": "^3.0.7",
-    "rollup-plugin-commonjs": "^9.1.5",
+    "rollup-plugin-babel": "^4.0.3",
+    "rollup-plugin-commonjs": "^9.1.6",
     "rollup-plugin-json": "^3.0.0",
     "rollup-plugin-multi-entry": "^2.0.2",
     "rollup-plugin-node-builtins": "^2.1.2",
-    "rollup-plugin-node-globals": "^1.2.1",
-    "rollup-plugin-node-resolve": "^3.3.0",
-    "rollup-plugin-uglify": "^3.0.0",
+    "rollup-plugin-node-globals": "^1.3.0",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-terser": "^2.0.2",
     "tap-merge": "^0.3.1",
-    "testem": "^2.9.2"
+    "testem": "^2.10.0"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.0.0"
```

Comment:
intentional for polly to work in node 6

Reply:
@jasonmit I think this should go in devDependencies